cmd/cerbosctl/put/internal/files/find.go

```diff
@@ -71,6 +78,52 @@ func Find(paths []string, recursive bool, fileType util.IndexedFileType, callbac
 	return nil
 }
 
+func fromZip(path string, fileType util.IndexedFileType, callback callback) error {
```

Comment:
Don't we already have this policy discovery logic defined elsewhere for a `fs.FS`? A `zip.Reader` is also a `fs.FS` so we should be able to avoid this duplication.

Reply:
I think the closest thing we have is [this method](https://github.com/cerbos/cerbos/blob/e7eda5e0787a566dabffa315f6c747edbe95f97b/internal/storage/index/builder.go#L82).
The problem with that method is it ignores the schemas, but in our case, we want to be aware of both policies and schemas.